docs/creating.md

```diff
@@ -15,13 +15,13 @@ project, the user will be prompted to fill in or confirm the default values.
 
 ## Minimal example
 
-```shell
-📁 my_copier_template ------------------------ # your template project
-├── 📄 copier.yml ---------------------------- # your template configuration
-├── 📁 .git ---------------------------------- # your template is a Git repository
-├── 📁 {{project_name}} ---------------------- # a folder with a templated name
-│   └── 📄 {{module_name}}.py.jinja ---------- # a file with a templated name
-└── 📄 {{_copier_conf.answers_file}}.jinja --- # answers are recorded here
+```tree result="shell"
+my_copier_template ---------------------------- # your template project
+    copier.yml -------------------------------- # your template configuration
+    .git/ ------------------------------------- # your template is a Git repository
+    {{project_name}} -------------------------- # a folder with a templated name
```

Comment:
I'd remove the ----

Reply:
Would you prefer having the `----` just removed, or should I use [annotations](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations) instead of the inline comments?

Reply:
Annotations would require lots of clicks :/
They're nice when you want to add links or other complex markup, but here I think it's better to keep simple comments.

Reply:
Done, see 2234129.